core/src/main/java/org/opensearch/sql/expression/operator/arthmetic/MathematicalFunction.java

```diff
@@ -181,20 +200,26 @@ private static DefaultFunctionResolver crc32() {
    */
   private static DefaultFunctionResolver euler() {
     return FunctionDSL.define(BuiltinFunctionName.E.getName(),
-        FunctionDSL.impl(() -> new ExprDoubleValue(Math.E), DOUBLE)
+            FunctionDSL.impl(() -> new ExprDoubleValue(Math.E), DOUBLE)
```

Comment:
Why did this function get reverted? Shouldn't it work with the base math function?

Reply:
The function was reverted because the format is different than that of other math functions, I believe it is because of the fact that euler doesn't take in any parameters so it is missing a section of code that all of the other functions have

Reply:
If that's the case, shouldn't the base function be able to handle it? Maybe have an optional parameter or a null can be passed it.

Reply:
The functions are so fundamentally different that a different base function would be required to be written for them, and as the only functions that would be able to use that new base function would be E and Pi I feel that it is currently unnecessary to do currently